util/linuxfw/iptables_runner.go

```diff
@@ -156,8 +156,9 @@ func (i *iptablesRunner) HasIPV6NAT() bool {
 	return i.v6NATAvailable
 }
 
-func isErrChainNotExist(err error) bool {
-	return errCode(err) == 1
+func isNotExistError(err error) bool {
+	e, ok := err.(*iptables.Error)
```

Comment:
errors.As instead? then it'll handle wrapped errors.

https://go.dev/blog/go1.13-errors